fix: init test context (#900)
vitest-dev/vitest: docs/guide/test-context.md

# Test Context

Inspired by [Playwright Fixtures](https://playwright.dev/docs/test-fixtures), Vitest's test context allows you to define utils, states, and fixtures that can be used in your tests.

## Usage

The first argument or each test callback is a test context.

```ts
import { it } from 'vitest'

it('should work', (ctx) => {
  // prints name of the test
  console.log(ctx.meta.name)
})
```

> ## Built-in Text Context

Reviewer: ```suggestion
## Built-in Test Context
```

#### `context.meta`

A readonly object containing metadata about the test.

#### `context.expect`

The `expect` API which bound to the current test.

## Extend Test Context

> The context are different for each test. You can access and extend them with in `beforeEach` and `afterEach` hook.

Reviewer: ```suggestion
The contexts are different for each test. You can access and extend them within the `beforeEach` and `afterEach` hooks.
```

```ts

> import { it } from 'vitest'

Reviewer: ```suggestion
import { it, beforeEach } from 'vitest'
```
